exercises/ex8/README.md

```diff
@@ -76,20 +76,36 @@ Instead of displaying the incidence data in a list, we want to display it in a `
 npm install chart.js --save-dev
 ```
 
-To get code completion support for Chart.js, you can install the type definitions from [DefinitelyTyped](https://www.npmjs.com/package/@types/chart.js):
+Now you can verify whether the custom middleware works fine or not by running the development server with `npm start` and trying to open the Chart.js file from [http://localhost:8080/resources/chart.js.js](http://localhost:8080/resources/chart.js.js)
 
-```sh
-npm install @types/chart.js --save-dev
+As `chart.js` includes its type definitions and as it's a standard node module, you now need to add the property `moduleResolution: "node"` to your `tsconfig.json`:
+
+```json
+{
+    "compilerOptions": {
+    [...]
+        "moduleResolution": "node",
+        "typeRoots": [
+            "./node_modules/@types",
+            "./node_modules/@openui5/ts-types-esm"
+        ],
+    [...]
+    }
+}
 ```
 
-Now you can verify whether the custom middleware works fine or not by running the development server with `npm start` and trying to open the Chart.js file from [http://localhost:8080/resources/chart.js.js](http://localhost:8080/resources/chart.js.js)
+This propery ensures that the TypeScript engine knows that types are available in the specified `typeRoots` and also in each node module.
+
+> **Remark:**
+> After adding this property to the `tsconfig.json` it may be necessary to restart your VSCode to resolve the type definitions properly.
```

Comment:
> restart your VSCode

or just restart the TS server? Probably you did not mention because it would have to be explained. Ok.

Reply:
Restart TS server option is gone in latest VSCode - now I may suggest to restart the language server?